main.py

```diff
@@ -1,10 +1,52 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for player, info in players_data.items():
+        email = info["email"]
+        bio = info["bio"]
+        races = info["race"]
+        skills = races["skills"]
+        guilds = info["guild"]
+
+        if not Race.objects.filter(name=races["name"]).exists():
+            Race.objects.create(
+                name=races["name"],
+                description=races["description"],
+            )
+        race = Race.objects.get(name=races["name"])
+
+        for skill in skills:
+            if not Skill.objects.filter(name=skill["name"]).exists():
+                Skill.objects.create(
+                    name=skill["name"],
+                    bonus=skill["bonus"],
+                    race=race
+                )
+        if guilds:
```

Comment:
I'd suggest to remove outer `if ... else` here and just assign the output of `create()` to `guilds` in the inner `if` block. By the way, why not name it `guild` right away?

Reply:
Thanks for ur advice, but i can't implement it correctly without errors. So let's wait for mentor review. ;)
 (`guild` name is fixed)

Reply:
At this point you can simply remove the `else` instruction. Since an empty `giuld` field is represented as `null` in json, it's already `None` if it's empty.